src/Nowruz/modules/settings/components/account/index.tsx

```diff
@@ -0,0 +1,140 @@
+import { AnyAction } from '@reduxjs/toolkit';
+import { useState } from 'react';
+import { Button } from 'src/Nowruz/modules/general/components/Button';
+import { FeaturedIcon } from 'src/Nowruz/modules/general/components/featuredIcon-new';
+import { Input } from 'src/Nowruz/modules/general/components/input/input';
+import { Modal } from 'src/Nowruz/modules/general/components/modal';
+import { SearchDropdown } from 'src/Nowruz/modules/general/components/SearchDropdown';
+import { deleteAccount } from 'src/pages/delete-profile/delete-profile.service';
+
+import css from './account.module.scss';
+
+
+
+const Account = () => {
+
+    const [modalVisibility, setModalVisibility] = useState(false);
+    let reasonbody = "";
+
+    const onChangeTextHandler = (e:AnyAction) => {
+        reasonbody = e.target.value;
+        console.log('gt',reasonbody);
+    };
+    
+    const closeAccount = () => {
+        deleteAccount(reasonbody);
+    };
+
+    return (
+        <>
+            <div className='hidden'>
+                <div className={css.borderSection}>
+                    <div className="flex flex-row w-full pt-8 items-center">
+                        <h2 className="grow css.title">Account Information</h2>
```

Comment:
I think css.title is not recognized here. Please use this
{ `grow ${css.title} `}